### PR TITLE
spec/steps/execute: Add execution of a given file.

### PR DESCRIPTION
**Pull request by thrix**
This should help use cases when people just want to run a script in the
test environment ...
    
Also should help with https://github.com/packit-service/packit-service/issues/281

Signed-off-by: Miroslav Vadkerti <mvadkert@redhat.com>

### CODE REVIEW

**Comment by thrix** [comment]
I am not still sure if this is a good idea, but leaving it here for discussion ...

File: spec/steps/execute.fmf [hunk 1]
```diff
@@ -67,3 +68,28 @@ example: |
...
+    /multi-file:
+        summary: Execute given local files
+        description:
+            Execute given local files on target system. The executor
+            copies the files to the target system (if applicable), sets
+            them as executable and executes them in a shell session one
+            after another. If the files have a relative path, it is relative
+            to the FMF root directory.
+        example: |
+            execute:
```
Comment (pvalena) [inline spec/steps/execute.fmf]: IMO this should be part of `Discover`. It has `how: shell` and `tests: ....`

**Comment by pvalena** [comment]
See here:
https://github.com/psss/tmt/pull/47/commits/c5546f6396805c70dab3a926d230d8cf5cf74bf9#diff-8012ed686caf41cde09689e803e7fa0c

File: spec/steps/execute.fmf (hunk 1 above)
Comment (thrix) [reply spec/steps/execute.fmf]: yeah :) I guess so, it was a late night false thought :D

**Comment by thrix** [comment]
Closing this, does not make much sense

**Pull request closed without merging**